tests/api/test_card_tokenization.py

```diff
@@ -0,0 +1,32 @@
+from authlib.oauth2.rfc6749 import OAuth2Token
+
+import pytest
+
+from littlepay.api.card_tokenization import CardTokenizationMixin
+
+
+@pytest.fixture
+def token():
+    return {"access_token": "1234"}
```

Comment:
Nitpick but this doesn't seem necessary, since the tests don't actually need to know anything specific about the token. 

The one in `conftest.py` is fine.

Reply:
👍 I can clean this up in a future PR